strider/fetcher.py

```diff
@@ -241,12 +242,18 @@ async def on_message(
         # process results
         for new_result in response["results"]:
             # queue the results for further processing
+            for nbs in new_result["node_bindings"].values():
+                for nb in nbs:
+                    nb["category"] = response["knowledge_graph"]["nodes"][nb["id"]]["category"]
             await self.put(merge_results([result, new_result]))
 
 
 def get_kp_request_body(
         qgraph: QueryGraph,
         edge: str,
```

Comment:
The step includes the edge, so we can simplify this set of function parameters slightly.